bip-sigsf.mediawiki

```diff
@@ -0,0 +1,194 @@
+<pre>
+  BIP: ???
+  Title: Modern soft-fork deployments
+  Author: ...
+  Comments-Summary: No comments yet.
+  Comments-URI: ???
+  Status: Proposed
+  Type: Informational
+  Created: ???
+  License: BSD-2-Clause
+</pre>
+
+==Abstract==
+
+This document specifies a proposed change to the semantics of the 'version' field in Bitcoin blocks, allowing multiple backward-compatible changes (further called "soft forks") to be deployed in parallel. It relies on interpreting the version field as a bit vector, where each bit can be used to track an independent change. These are tallied each retarget period. Once the consensus change succeeds or fails, the bit can be reused for later changes.
+
+==Motivation==
+
```

Comment:
abcd

Reply:
amiti's buggy comment debugging?

Reply:
I think you mean _brilliant_

Reply:
debugging buggy comments makes sense, why would you debug brilliant comments?